nixos/modules/services/networking/firefox/sync-server.nix

```diff
@@ -124,7 +124,13 @@ in
       description = "Firefox Sync Server";
       wantedBy = [ "multi-user.target" ];
       path = [ pkgs.pythonPackages.pasteScript pkgs.coreutils ];
-      environment.PYTHONPATH = "${pkgs.pythonPackages.syncserver}/lib/${pkgs.pythonPackages.python.libPrefix}/site-packages";
+      environment.PYTHONPATH = let
+          penv = pkgs.python.buildEnv.override {
+            extraLibs = [ pkgs.pythonPackages.syncserver ];
+            ignoreCollisions = true;
+          };
+        in "${penv}/${pkgs.python.sitePackages}/";
```

Comment:
> But when do they end up in one closure then? Since if I just build python.pkgs.syncserver, they don't end up in the same closure.

So here I was checking with
```
nix-store -qR $(nix-build -A python.pkgs.syncserver) | grep wsgi
```
but should have checked for `WSGI` instead.

Its also interesting that `catch_conflicts.py` didn't catch this when building  `syncserver`, but that's because that script checks package names, and not the module names.

> I agree, but I think the problem does not come from nixpkgs, it comes from the dependencies of syncserverstorage which we can't really fix ourselves

Exactly. But apparently `syncserver` and `paster` do seem to work using the same `wsgiproxy`, which is good news. 

https://github.com/FRidh/nixpkgs/commit/64ef1e7c020a4b2b0cca1b9c6aa87970b1682b6b